functional/basic-attestation-on-localhost/test.sh

```diff
@@ -118,7 +118,19 @@ rlJournalStart
         rlRun "limeWaitForVerifier"
         rlRun "limeStartRegistrar"
         rlRun "limeWaitForRegistrar"
-        rlRun "limeStartAgent"
+        # set service environment variables
+        if [ -f /usr/lib/systemd/system/keylime_agent.service -o -f /etc/systemd/system/keylime_agent.service ]; then
```

Comment:
I am thinking whether it won't be better to move this kind of setup to test-helpers library itself, somewhere here
https://github.com/RedHat-SP-Security/keylime-tests/blob/3de7e180836511a2a1f0ecd4cd95032611225726/Library/test-helpers/lib.sh#L1299

Reply:
Yes, it would be better if this initialization was centralized and we could start the agent with a simple `limeStartAgent` like other services.